alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Would be good to link to alembic docs and say that we're using it so that tables can change in production without losing of any data. Also worth updating the module level readme (for imaging-api, assuming we move it there)